docker-build/Dockerfile

```diff
@@ -1,57 +1,28 @@
-FROM        debian 
-
-ARG         gsd
-ENV         GITHUB_STABLE_DIFFUSION $gsd
-
-ARG         rsd
-ENV         REQS $rsd
-
-ARG         cs
-ENV         CONDA_SUBDIR $cs
-
-ENV         PIP_EXISTS_ACTION="w"
-
-# TODO: Optimize image size
-
-SHELL       ["/bin/bash", "-c"]
-
-WORKDIR     /
-RUN         apt update && apt upgrade -y \
-            && apt install -y \
-            git \
-            libgl1-mesa-glx \
-            libglib2.0-0 \
-            pip \
-            python3 \
-            && git clone $GITHUB_STABLE_DIFFUSION
-
-# Install Anaconda or Miniconda
-COPY        anaconda.sh .
-RUN         bash anaconda.sh -b -u -p /anaconda && /anaconda/bin/conda init bash
-
-# SD 
-WORKDIR     /stable-diffusion 
-RUN         source ~/.bashrc \
-            && conda create -y --name ldm && conda activate ldm \
-            && conda config --env --set subdir $CONDA_SUBDIR \
-            && pip3 install -r $REQS \   
-            && pip3 install basicsr facexlib realesrgan \          
-            && mkdir models/ldm/stable-diffusion-v1 \
-            && ln -s "/data/sd-v1-4.ckpt" models/ldm/stable-diffusion-v1/model.ckpt 
-
-# Face restoreation
-# by default expected in a sibling directory to stable-diffusion
-WORKDIR     /       
-RUN         git clone https://github.com/TencentARC/GFPGAN.git 
-
-WORKDIR     /GFPGAN 
-RUN         pip3 install -r requirements.txt \
-            && python3 setup.py develop \
-            && ln -s "/data/GFPGANv1.4.pth" experiments/pretrained_models/GFPGANv1.4.pth
-
-WORKDIR     /stable-diffusion 
-RUN         python3 scripts/preload_models.py         
-
-WORKDIR     /
-COPY        entrypoint.sh .
-ENTRYPOINT ["/entrypoint.sh"]
\ No newline at end of file
+FROM nvidia/cuda:11.7.1-base-ubuntu22.04
+FROM continuumio/miniconda3
+
+RUN apt update
+RUN apt install dos2unix -y
+
+WORKDIR /
+COPY . /InvokeAI/
+
+WORKDIR /InvokeAI
+
+RUN mkdir -p models/ldm/stable-diffusion-v1
+#RUN pwd
+#RUN ls -la
+RUN mv model.ckpt models/ldm/stable-diffusion-v1/model.ckpt
+
+RUN conda env create -f environment.yml
+RUN conda env list
+SHELL ["conda", "run", "-n", "invokeai", "/bin/bash", "-c"]
+
+RUN python3 scripts/preload_models.py
+
+#WORKDIR /
+#ADD /docker-build/start.sh .
+RUN dos2unix docker-build/entrypoint.sh
+RUN chmod +x docker-build/entrypoint.sh
+
+ENTRYPOINT docker-build/entrypoint.sh
```

Comment:
Why not move the entrypoint.sh to the container root (or to (/usr/bin/entrypoint.sh))

Reply:
I wasn't thinking about it, assuming someone will invoke it form a command line makes sense though so I can add something to move it to the root.